xbmc/windowing/gbm/drm/DRMAtomic.cpp

```diff
@@ -60,17 +46,13 @@ uint32_t GetScalingFactor(uint32_t srcWidth,
 
 bool CDRMAtomic::SetScalingFilter(struct CDRMObject* object, const char* name, const char* type)
 {
-  uint32_t filter_type = GetScalingFilterType(type);
-  if (object->SupportsPropertyAndValue(name, filter_type))
+  uint64_t filter_type = GetScalingFilterType(type);
```

Comment:
Does this even build? How can you say you tested this if it doesn't build?

Reply:
My bad, some garbage code left before pushing into PR, I will fix it. Thanks for pointing it out.